### PR TITLE
[picker] upgrade to 2.1.0

### PR DESCRIPTION
**Pull request by ajsmth**
# Why

update `@react-native-picker/picker` to `2.0.0`

# How

`et uvm -m @react-native-picker/picker`

# Test Plan

NCL 

# Checklist

- [x] Documentation is up to date to reflect these changes (eg: https://docs.expo.io and README.md).
- [x] This diff will work correctly for `expo build` (eg: updated `@expo/xdl`).
- [x] This diff will work correctly for `expo prebuild` & EAS Build (eg: updated a module plugin).

### CODE REVIEW

**Review by tsapeta — CHANGES_REQUESTED** [review]
Update version in changelog, re-yarn and reinstall pods 😉

**Comment by ajsmth** [comment]
> Update version in changelog, re-yarn and reinstall pods 😉

🤦

File: apps/bare-expo/ios/Podfile.lock [hunk 1]
```diff
@@ -1204,7 +1204,7 @@ SPEC CHECKSUMS:
   AppAuth: 31bcec809a638d7bd2f86ea8a52bd45f6e81e7c7
   boost-for-react-native: 39c7adb57c4e60d6c5479dd8623128eb5b3f0f2c
   CocoaLumberjack: b7e05132ff94f6ae4dfa9d5bce9141893a21d9da
-  DoubleConversion: cf9b38bf0b2d048436d9a82ad2abe1404f11e7de
+  DoubleConversion: cde416483dac037923206447da6e1454df403714
```
Comment (tsapeta) [inline apps/bare-expo/ios/Podfile.lock:1207]: This checksum is the incorrect one 😄 Same for glog, so you should do clean install

Comment (ajsmth) [reply apps/bare-expo/ios/Podfile.lock]: 🤦 🤦 double

**Review by tsapeta — APPROVED** [review]

**Comment by brentvatne** [comment]
it looks like 2.1.0 was released a couple days ago and includes a few fixes for web. should be easy to pull in :)

File: apps/bare-expo/ios/Podfile.lock [hunk 2]
```diff
@@ -660,7 +660,7 @@ PODS:
     - React-Core
   - RNCMaskedView (0.2.5):
     - React-Core
-  - RNCPicker (1.16.7):
+  - RNCPicker (2.0.0):
```
Comment (brentvatne) [inline apps/bare-expo/ios/Podfile.lock:663]: need to run pod install again

File: yarn.lock [hunk 3]
```diff
@@ -5716,10 +5716,10 @@
   resolved "https://registry.yarnpkg.com/@react-native-masked-view/masked-view/-/masked-view-0.2.5.tgz#2505f269a53b134e3306a2772d981138e11f1537"
   integrity sha512-yXaWyV8xu1N5tfdfP8L29U0TIDvBriN4zHUI+Zs7Sw+VI8pBbEEYoITK9aFt495Y0/PPm4FsTpa35hOeTLfGPA==
 
-"@react-native-picker/picker@1.16.7":
-  version "1.16.7"
-  resolved "https://registry.yarnpkg.com/@react-native-picker/picker/-/picker-1.16.7.tgz#2c943572efcd21c218ee03f23ba1475544b24cf1"
-  integrity sha512-/QJxL4JniloFvdPDufQTiMQQbpucCQ0w7wvGhZQt0x3KVxvI1f/vzDO+669r/1yipcFZbSLcYIiQSMSpZtkqiQ==
+"@react-native-picker/picker@2.0.0":
```
Comment (brentvatne) [inline yarn.lock:5719]: need to run yarn again with 2.1.0

**Review by brentvatne — APPROVED** [review]
just need to update lockfiles then should be good to go

**Pull request merged**